### PR TITLE
tasks + tests 5

### PR DESCRIPTION
**Pull request by muhaale**

### CODE REVIEW

File: tests/anmykh/test_homework_5.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ({"a": 123, "b": 456, "c": 789}, dict),
+    ({"c": 789, "a": 123, "b": 456}, dict),
+    ({"a": 1, 2: "b",
+      None: "abc",
+      (1, "a"): "some",
+      True: "kek",
+      frozenset((1, 2)): "lol"}, dict)
+])
+def test_is_dict(param, data_type):
+    assert isinstance(param, data_type)
```
Comment (kudinov-fedor) [inline tests/anmykh/test_homework_5.py:15]: be carefull with identations:

```
@pytest.mark.parametrize(["param", "data_type"], [
    ({"a": 123, "b": 456, "c": 789}, dict),
    ({"a": 123, "b": 456, "c": 789}, dict),
    ({"c": 789, "a": 123, "b": 456}, dict),
    ({"a": 1, 2: "b",
      None: "abc",
      (1, "a"): "some",
      True: "kek",
      frozenset((1, 2)): "lol"}, dict)
])
def test_is_dict(param, data_type):
    assert isinstance(param, data_type)

```

File: tests/anmykh/test_homework_5.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {"a": "1", "b": "2"}, "a", '1'),
+        ({"c": "a", "a": "b"}, "c", 'a'),
+        ({"m": None, "v": None}, "m", None)])
+def test_update_dict(dict_collection, key, result):
+    assert dict_collection.get(key) == result
+
+
+def error_func():
+    with pytest.raises(ArithmeticError):
+        a = 5 / 0
```
Comment (kudinov-fedor) [inline tests/anmykh/test_homework_5.py:28]: minimum code to fail:


```
def error_func():
    with pytest.raises(ArithmeticError):
        5 / 0

```

File: tests/anmykh/test_homework_5.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+def error_func_2():
+    with pytest.raises(TypeError):
+        a = []
+        a[0]
+
+
+def error_func_3():
+    with pytest.raises(NameError):
+    x
```
Comment (kudinov-fedor) [inline tests/anmykh/test_homework_5.py:39]: wrong identation, this should fail due to empty context manager body

File: tests/anmykh/test_homework_5.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+def error_func():
+    with pytest.raises(ArithmeticError):
+        a = 5 / 0
+
+
+def error_func_2():
+    with pytest.raises(TypeError):
+        a = []
+        a[0]
```
Comment (kudinov-fedor) [inline tests/anmykh/test_homework_5.py:34]: a = [].     сan be moved from with block

**Review by kudinov-fedor — CHANGES_REQUESTED** [review]
need to fix some comments

**Review by fkudinov-mn — APPROVED** [review]

**Review by kudinov-fedor — APPROVED** [review]

**Pull request merged**